app/javascript/products.vue

```diff
@@ -0,0 +1,171 @@
+<template lang="pug">
+  .page-body
+    .container(v-if="loaded")
+      nav.pagination.is-top
+        pager-top(
+          v-if="totalPages > 1 && products.length > 0"
+          v-model='currentPage'
+          :page-count="totalPages"
+          :page-range=5
+          :prev-text="`<i class='fas fa-angle-left'></i>`"
+          :next-text="`<i class='fas fa-angle-right'></i>`"
+          :first-button-text="`<i class='fas fa-angle-double-left'></i>`"
+          :last-button-text="`<i class='fas fa-angle-double-right'></i>`"
+          :click-handler="paginateClickCallback"
+          :container-class="'pagination__items'"
+          :page-class="'pagination__item'"
+          :page-link-class="'pagination__item-link'"
+          :disabled-class="'pager-disable'"
+          :active-class="'is-active'"
+          :prev-class="'is-prev pagination__item'"
+          :prev-link-class="'is-prev pagination__item-link'"
+          :next-class="'is-next pagination__item'"
+          :next-link-class="'is-next pagination__item-link'"
+          :first-last-button="true"
+          :hide-prev-next="true"
+          :margin-pages="0"
+          :break-view-text="null"
+        )
+      .thread-list.a-card(v-if="products.length > 0")
+        product(v-for="(product, index) in products"
```

Comment:
index利用されていなさそうなので、削除できますかね？？
```suggestion
        product(v-for="product in products"
```

Reply:
削除しました！

fb44480